src/js/modules/core/components/DataErrorPage.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Grid } from "react-bootstrap/lib";
+import injectSheet from "react-jss";
+import { Helmet } from "react-helmet";
+
+const styles = {
+  DataErrorPage: {
+    fontFamily: "Minion Pro",
+    paddingTop: "80px",
+    textAlign: "center",
+    "& h1": {
+      fontFamily: "Canela",
+      fontSize: "36px",
+      lineHeight: "40px",
+    },
+    "& p": {
+      fontSize: "21px",
+      lineHeight: "27px",
+      marginBottom: "2px",
+    },
+  },
+  retryButton: {
+    backgroundColor: "#3472b7",
+    border: "1px solid #3472b7",
+    borderRadius: "3px",
+    color: "#fff",
+    fontSize: "15px",
+    fontStyle: "italic",
+    height: "32px",
+    marginTop: "15px",
+    textAlign: "center",
+    width: "70px",
+  },
+};
+
+const DataErrorPage = ({ classes, error, action}) => {
+  console.error(error);
```

Comment:
Ehhh

Reply:
It's so we developers get a hint of what goes on without `redux-logger`.

Reply:
But how will you? You're not sitting at the user's computer